pages/api/id/[id].js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { ingredientsWithMeasure } from '../../../util/functions';
+
+export default async function getRecipeDetailsById(req, res) {
+  const { id } = req.query
+  const response = await axios.get(`https://www.themealdb.com/api/json/v1/1/lookup.php?i=${id}`);
+  const recipe = response.data.meals[0];
```

Comment:
can we verify that meals is an array before accessing its items? we should guarantee that this would not blow up if meals is undefined

Reply:
I had already verified this